pkg/standalone-utils/contracts/PauseHelper.sol

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+pragma solidity ^0.8.24;
+
+import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+
+import { IVault } from "@balancer-labs/v3-interfaces/contracts/vault/IVault.sol";
+import { IVaultAdmin } from "@balancer-labs/v3-interfaces/contracts/vault/IVaultAdmin.sol";
+import { SingletonAuthentication } from "@balancer-labs/v3-vault/contracts/SingletonAuthentication.sol";
+
+contract PauseHelper is SingletonAuthentication {
+    using EnumerableSet for EnumerableSet.AddressSet;
+
+    /**
+     * @notice Revert if the pool is already in the list of pools
+     * @param  pool Pool that tried to be added
+     */
+    error PoolExistInPausableSet(address pool);
+
+    /**
+     * @notice Revert if the pool is not in the list of pools
+     * @param  pool Pool that not found
+     */
+    error PoolNotFoundInPausableSet(address pool);
+
+    /// @notice An index is beyond the current bounds of the set.
+    error IndexOutOfBounds();
+
+    /**
+     * @notice Emitted when a pool is added to the list of pools that can be paused
+     * @param pool Pool that was added
+     */
+    event PoolAddedToPausableSet(address pool);
+
+    /**
+     * @notice Emitted when a pool is removed from the list of pools that can be paused
+     * @param pool Pool that was removed
+     */
+    event PoolRemovedFromPausableSet(address pool);
+
+    EnumerableSet.AddressSet private _poolSet;
+
+    constructor(IVault vault) SingletonAuthentication(vault) {
+        // solhint-disable-previous-line no-empty-blocks
+    }
+
+    /***************************************************************************
+                                                      Manage Pools
+    ***************************************************************************/
+
+    /**
+     * @notice Add pools to the list of pools that can be paused.
+     * @dev This is a permissioned function. Only authorized accounts (e.g., monitoring service providers) may add
+     * pools to the pause list.
+     * @param newPools List of pools to add
```

Comment:
```suggestion
     * pools to the pause list.
     *
     * @param newPools List of pools to add
```
Convention is to insert a blank line if the dev tag is multi-line. Same below.

Periods after notice and dev; no periods on param and return